bindings/python/pinocchio/__init__.py

```diff
@@ -14,12 +14,12 @@
 from .deprecated import *
 from .shortcuts import *
 
-if pin.WITH_FCL:
+if pin.WITH_HPP_FCL:
   try:
     import hppfcl as fcl
```

Comment:
I suggest to replace `import hppfcl as fcl` to just `import hppfcl` too. I thought this is what @jmirabel meant

Reply:
Also, I would like to point out that this breaks existing Python code using the old Pinocchio-integrated HPP-FCL bindings: `pinocchio.CollisionGeometry` will not work, for instance.
In order to avoid that, you should import in Pinocchio the old classes one by one. If you want to keep this behavior forever, you can just leave it like that. Otherwise, you can easily make the pinocchio-integrated version deprecated

Reply:
Done!